File: JavaScript/Faccat/Exercicio6b.js
const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question('Digite o valor da base: ', function(base) {
  rl.question('Digite o valor da altura: ', function(altura) {
    const area = (parseFloat(base) * parseFloat(altura)) / 2;
    console.log('A área do triângulo é: ' + area);

    rl.close();
  });
});

File: Java/src/Faccat/Exercicio6b.java
package Faccat;

import java.util.Scanner;

public class Exercicio6b {
    public static void main(String[] args) {
        double base, altura, area;
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite o valor da base: ");
        base = scanner.nextDouble();

        System.out.print("Digite o valor da altura: ");
        altura = scanner.nextDouble();

        area = (base * altura) / 2;

        System.out.println("A área do triângulo é: " + area);

        scanner.close();
    }
}







